translations/ca/project.yml

```diff
@@ -23,6 +23,7 @@ project-help-license: 'Necessito ajuda per definir els retorns col·lectius i le
 project-hide-donors: 'Amagar cofinançadors'
 project-hide-needs: 'Amagar llistat de necessitats'
 project-invest: Aportació
+project-invest-from-subscription: Suscripció
```

Comment:
This one is [subscripció](https://dlc.iec.cat/Results?IdE=0081896&DecEntradaText=subscripci%C3%)